test/coverlet.collector.tests/CoverletSettingsParserTests.cs

```diff
@@ -62,22 +78,68 @@ public void ParseShouldCorrectlyParseConfigurationElement()
             Assert.Equal("[coverlet.*]*", coverletSettings.ExcludeFilters[0]);
+            Assert.Equal("[coverlet.*.tests?]*", coverletSettings.ExcludeFilters[1]);
+            Assert.Equal("[coverlet.*.tests.*]*", coverletSettings.ExcludeFilters[2]);
+            
             Assert.False(coverletSettings.UseSourceLink);
             Assert.True(coverletSettings.SingleHit);
             Assert.True(coverletSettings.IncludeTestAssembly);
         }
 
+        [Fact]
+        public void ParseShouldCorrectlyParseConfigurationElementWithNullInnerText()
+        {
+            var testModules = new List<string> { "abc.dll" };
+            var doc = new XmlDocument();
+            var configElement = doc.CreateElement("Configuration");
+            this.CreateCoverleteNullInnerTextNodes(doc, configElement, CoverletConstants.IncludeFiltersElementName);
+            this.CreateCoverleteNullInnerTextNodes(doc, configElement, CoverletConstants.ExcludeFiltersElementName);
+            this.CreateCoverleteNullInnerTextNodes(doc, configElement, CoverletConstants.IncludeDirectoriesElementName);    
+            this.CreateCoverleteNullInnerTextNodes(doc, configElement, CoverletConstants.ExcludeSourceFilesElementName);
+            this.CreateCoverleteNullInnerTextNodes(doc, configElement, CoverletConstants.ExcludeAttributesElementName);
+
+            CoverletSettings coverletSettings = _coverletSettingsParser.Parse(configElement, testModules);
+
+            Assert.Equal("abc.dll", coverletSettings.TestModule);
+            Assert.Empty(coverletSettings.IncludeFilters);
+            Assert.Empty(coverletSettings.IncludeDirectories);
+            Assert.Empty(coverletSettings.ExcludeSourceFiles);
+            Assert.Empty(coverletSettings.ExcludeAttributes);
+            Assert.Single(coverletSettings.ExcludeFilters, "[coverlet.*]*");
+        }
+
+        [Fact]
+        public void ParseShouldCorrectlyParseConfigurationElementWithNullElements()
+        {
+            var testModules = new List<string> { "abc.dll" };
+            var doc = new XmlDocument();
+            var configElement = doc.CreateElement("Configuration");
+
+            CoverletSettings coverletSettings = _coverletSettingsParser.Parse(configElement, testModules);
+
+            Assert.Equal("abc.dll", coverletSettings.TestModule);
+            Assert.Null(coverletSettings.IncludeFilters);
+            Assert.Null(coverletSettings.IncludeDirectories);
+            Assert.Null(coverletSettings.ExcludeSourceFiles);
+            Assert.Null(coverletSettings.ExcludeAttributes);
+            Assert.Single(coverletSettings.ExcludeFilters, "[coverlet.*]*");
+        }
+
         [Theory]
         [InlineData(" , json", 1, new[] { "json" })]
         [InlineData(" , json, ", 1, new[] { "json" })]
         [InlineData("json,cobertura", 2, new[] { "json", "cobertura" })]
+        [InlineData("json,\r\ncobertura", 2, new[] { "json", "cobertura" })]
```

Comment:
Please add some space and tabs also here

Reply:
Added a few cases with space, tabs and a mix of everything